python/lsst/pipe/tasks/matchFakes.py

```diff
@@ -157,10 +220,12 @@ def _trimFakeCat(self, fakeCat, image):
             The catalog of fake sources to be input
         image : `lsst.afw.image.exposure.exposure.ExposureF`
             The image into which the fake sources should be added
+        skyMap : `lsst.skymap.SkyMap`
+            SkyMap defining the tracts and patches the fakes are stored over.
 
         Returns
         -------
-        fakeCat : `pandas.core.frame.DataFrame`
+        fakeCats : `pandas.core.frame.DataFrame`
```

Comment:
Please explain why this has been pluralized in the description for the parameter.